### PR TITLE
readme

### PR DESCRIPTION
**Pull request by marco-tranzatto**

### CODE REVIEW

**Review by alexmillane — APPROVED** [review]
Thanks marco. This is great. LGTM

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
...
+This package is a catkinized version of the original library [ORB_SLAM2](https://github.com/raulmur/ORB_SLAM2)
+
+**Keywords:** CATKIN, ORB_SLAM2, SLAM
+
+### License
+
+The source code is released under a [TODO @alexmillane insert license](put_link_here).
+
+**Author(s): Alex Millane
+Maintainer: Alex Millane
```
Comment (alexmillane) [inline README.md]: Oh and add yourself to these lists!.. If you want.

Comment (marco-tranzatto) [reply README.md]: nope!

Comment (marco-tranzatto) [reply README.md]: Also, can you please add a license to his before merging in?

**Pull request merged**